scripts/run_e2e_tests.py

```diff
@@ -269,7 +269,10 @@ def run_tests(args):
             log_level=args.server_log_level,
             clear_datastore=True,
             skip_sdk_update_check=True,
-            env={'PORTSERVER_ADDRESS': common.PORTSERVER_SOCKET_FILEPATH}))
+            env={
+                # **os.environ,
```

Comment:
What's this comment for? (If it's actually meant to be a comment, maybe elaborate?)

Reply:
I was testing if the code works without this or not. Removed the comment.